File: Msys_Programs/MSys_Projects/PythonProg/Python_DSA/Binary_Search/Linear_Search.py
def linearsearch(A, key):
    index = 0
    while index < len(A):
        if A[index] == key:
            return index
        index += 1
    return -1

A = [65,25,36,35,22,99,202]
# b = linearsearch(A, 35)
b = linearsearch(A, 100)
print("result:", b)

File: Msys_Programs/MSys_Projects/GoProg/DataStructure/Binary_Search/Linear_Search.go
package main

import "fmt"

func linearsearch(A []int, key int) int {
	index := 0
	for index < len(A) {
		if A[index] == key {
			return index
		}
		index += 1
	}
	return -1
}

func main() {
	A := []int{25, 36, 65, 96, 88}
	key := 36
	fmt.Println("Original list and key:", A)
	fmt.Println("Original key:", key)

	fmt.Println("Index No:", linearsearch(A, key))

}
